test/test_core.py

```diff
@@ -7678,14 +7678,14 @@ def test_embind(self, args):
     ''')
     self.do_runf('test_embind.cpp', 'abs(-10): 10\nabs(-11): 11', emcc_args=args)
 
-  @parameterized({
-    '': ([],),
-    'pthreads': (['-pthread', '-sPROXY_TO_PTHREAD', '-sEXIT_RUNTIME'],),
-  })
   @node_pthreads
-  def test_embind_2(self, args):
+  def test_embind_2(self):
     self.maybe_closure()
-    self.emcc_args += ['-lembind', '--post-js', 'post.js'] + args
+    self.emcc_args += [
+      '-lembind', '--post-js', 'post.js',
+      # for extra coverage, test using pthreads
+      '-pthread', '-sPROXY_TO_PTHREAD', '-sEXIT_RUNTIME'
+    ]
     create_file('post.js', '''
       function printLerp() {
```

Comment:
Another solution would be to skip the `@node_pthreads` and add this to the test body:

```
if args:
  self.setup_node_pthreads()
```

Another solution would be to have `@node_pthreads` no longer automatically add `-pthread` to the emcc_args.. which seem like maybe a good idea anyway.

Or we can just land this if you really feel like the two variants are not adding anything.

Reply:
Yeah, that's my sense: there's nothing special here regarding pthreads, it's just more coverage. So a single test as in the PR seems enough.

Reply:
(That is, we were wasteful before, is my point, and this saves some waste.)